ngraph/frontend/ir/src/ir_deserializer.cpp

```diff
@@ -0,0 +1,745 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
which tests cover this new code? 'ieFuncTests'? I see few 'serialization tests' but not sure if this amount of code is covered only by these 2 tests

Reply:
This amount of code is not new. This code was marked as changed because I've split the giant file with all code to the smaller files. So all this deserialization engine was copy-pasted as is. The only change here is rt_info support.